docs/plaindate.md

```diff
@@ -87,31 +87,33 @@ Additionally, if the result is earlier or later than the range of dates that `Te
 Example usage:
 
 ```javascript
-date = Temporal.PlainDate.from('2006-08-24'); // => 2006-08-24
-date = Temporal.PlainDate.from('2006-08-24T15:43:27'); // => 2006-08-24
-date = Temporal.PlainDate.from('2006-08-24T15:43:27Z'); // => 2006-08-24
+date = Temporal.PlainDate.from('2006-08-24'); // => '2006-08-24'
+date = Temporal.PlainDate.from('2006-08-24T15:43:27'); // => '2006-08-24'
+date = Temporal.PlainDate.from('2006-08-24T15:43:27Z'); // => '2006-08-24'
 date = Temporal.PlainDate.from('2006-08-24T15:43:27+01:00[Europe/Brussels]');
-  // => 2006-08-24
-date === Temporal.PlainDate.from(date) // => true
+  // => '2006-08-24'
+date.equals(Temporal.PlainDate.from(date)); // => true
```

Comment:
I think we want to demonstrate that it always creates a new object:

```suggestion
date === Temporal.PlainDate.from(date) // => false
```

Reply:
Got it, will change in all such locations.